spec/rubocop/cop/rspec/hook_argument_spec.rb

```diff
@@ -62,6 +49,13 @@
         config.after(:each)  { true }
         ^^^^^^^^^^^^^^^^^^^ Omit the default `:each` argument for RSpec hooks.
       RUBY
+
+      expect_correction(<<-RUBY)
+        before { true }
+        after  { true }
+        around { true }
+        config.after  { true }
```

Comment:
Cosmetic: extra space, in the "bad" example as well.

Reply:
Fixed